src/lib/translators/mongo4.ts

```diff
@@ -44,6 +44,14 @@ function transformToDate(step: Readonly<ToDateStep>): MongoStep[] {
     format = format.replace('%B', '%b');
   }
 
+  const input = {
+    $cond: [
+      { $eq: [{ $type: $$(step.column) }, 'int'] },
```

Comment:
@austil has a good point, we could implement here the same heuristic as we added in pandas.
Meaning:
```
$cond: [
  { $and: [ { $eq: [{ $type: $$(step.column) }, 'int'] }, { [step.column]: { $lt: 10_000 } } ] }
```

That may ensure same results for both backends regarding timestamps. What do you thinks ?

Reply:
Alright je corrige ça :pray:

Reply:
https://github.com/ToucanToco/weaverbird/pull/1286/commits/8d97f24cad1fbcd4614eb1e49869db2d8ad7e09c